gradle/examples/spring_boot/Dockerfile

```diff
@@ -1,3 +0,0 @@
-FROM openjdk:8-jdk
```

Comment:
Since you removed this file, I think it makes sense to remove the corresponding entry in `examples/spring_boot/cloudbuild.yaml` too.
```yaml
steps:
- name: 'gcr.io/cloud-builders/gradle'
  args: ['build']
- name: 'gcr.io/cloud-builders/docker'
  args: ['build', '--tag=gcr.io/$PROJECT_ID/spring-boot', '.']
images: ['gcr.io/$PROJECT_ID/spring-boot']
```